client/state/comments/constants.js

```diff
@@ -6,6 +6,13 @@ export const PLACEHOLDER_STATE = {
 	ERROR: 'ERROR',
 };
 
+// values conveniently also correspond to css classNames to apply
+export const POST_COMMENT_DISPLAY_TYPES = {
+	singleLine: 'is-single-line',
+	excerpt: 'is-excerpt',
+	full: 'is-full',
+};
+
 export const APPROVED_STATUS = 'approved';
```

Comment:
I'd start with `STATUS_` here for consistency (e.g. `STATUS_APPROVED`, `STATUS_DISAPPROVED`).

Reply:
looks like its not used anywhere in calypso....i can remove them completely